doc/pages/tutorials/010-astarte_in_5_minutes.md

```diff
@@ -203,7 +203,7 @@ $ astartectl appengine devices get-samples <your device id> org.astarte-platform
 
 If you get a meaningful value, congratulations - you have a working Astarte installation with your first `datastream` coming in!
 
-Moreover, Astarte's Docker Compose also installs [Astarte Dashboard](https://github.com/astarte-platform/astarte-dashboard), from which you can manage your Realms and install Triggers, Interfaces and more from a Web UI. It is accessible by default at `http://localhost:4040/` - remember that if you are not exposing Astarte from `localhost`, you have to change Realm Management API's URL in Dashboard's configuration file, to be found in `compose/astarte-dashboard/config.json` in Astarte's repository. You can generate a token for Astarte Dashboard, as usual, through `astartectl utils gen-jwt realm-management -k test_private.pem`. Grant a longer expiration by using the `-e` parameter to avoid being logged out too quickly.
+Moreover, Astarte's Docker Compose also installs [Astarte Dashboard](https://github.com/astarte-platform/astarte-dashboard), from which you can manage your Realms and install Triggers, Interfaces and more from a Web UI. It is accessible by default at `http://localhost:4040/` - remember that if you are not exposing Astarte from `localhost`, you have to change Realm Management API's URL in Dashboard's configuration file, to be found in `compose/astarte-dashboard/config.json` in Astarte's repository. You can generate a token for Astarte Dashboard, as usual, through `astartectl utils gen-jwt all-realm-apis -k test_private.pem -e 0`. Optionally set a specific expiration by using the `-e <seconds>` parameter, here `-e 0` means it doesn't expire.
```

Comment:
generating never expiring tokens should be avoided when possible as there is no mechanism to invalidate a token without changing the realm private key.

I think it's better handled by astartectl, see also:
https://github.com/astarte-platform/astartectl/pull/128

Reply:
That's brilliant! I added a reference to the 8 hours default and removed `-e 0`.